README.md

```diff
@@ -152,19 +152,21 @@ Most methods and properties from `pathlib.Path` are supported except for the one
 | `with_suffix`          | ✅                | ✅         | ✅         |
 | `write_bytes`          | ✅                | ✅         | ✅         |
 | `write_text`           | ✅                | ✅         | ✅         |
-| `absolute`             | ❌                | ❌         | ❌         |
+| `absolute`             | ✅                | ✅         | ✅         |
+| `resolve`              | ✅                | ✅         | ✅         |
+| `relative_to`          | ✅                | ✅         | ✅         |
+| `is_absolute`          | ✅                | ✅         | ✅         |
+| `is_relative_to`       | ✅                | ✅         | ✅         |
```

Comment:
I think these are ordered by implemented/not-implemented then alphabetical within that—would you mind updating? Looks like `python docs/make_support_table.py` may print one out that you can just copy-paste.